developerportal/templates/organisms/filter-list.html

```diff
@@ -1,54 +1,88 @@
     </div>
-    <div class="mzp-l-content">
-        {% if heading %}
-          <h2>{{ heading }}</h2>
+  </aside>
+  <div class="mzp-l-main">
+    <div class="mzp-l-card-half" id="{{ type }}-cards">
+      {% for resource in resources %}
+        {% if type == "article" %}
+          {% include "molecules/cards/card.html" with resource=resource.article show_author=True %}
+        {% elif type == "article_or_video" %}
+          {% if resource.video %}
+            {% include "molecules/cards/card.html" with resource=resource.video %}
+          {% else %}
+            {% include "molecules/cards/card.html" with resource=resource.article show_author=True %}
+          {% endif %}
+        {% elif type == "person" %}
+          {% include "molecules/cards/card.html" with resource=resource.person %}
+        {% elif type == "event" %}
+          {% include "molecules/cards/card.html" with resource=resource.event %}
         {% endif %}
-        <div class="filter-list-content">
-          <aside class="filter-list-sidebar">
-            <details class="filter-list-sidebar-content filter-list-sidebar-content-mobile">
-              <summary>Filter</summary>
-              {% include "molecules/filter-form.html" with resources=resources filters=filters %}
-            </details>
-            <div class="filter-list-sidebar-content filter-list-sidebar-content-desktop">
-              {% include "molecules/filter-form.html" with resources=resources filters=filters %}
-            </div>
-          </aside>
-          <div class="filter-list-items">
-            <div class="{% if type != 'event' %}card-mzp-l-content{% endif %}" id="{{ type }}-cards">
-              {% for resource in resources %}
-                {% if type == "article" %}
-                  {% include "molecules/cards/card.html" with resource=resource.article show_author=True %}
-                {% elif type == "article_or_video" %}
-                  {% if resource.video %}
-                    {% include "molecules/cards/card.html" with resource=resource.video %}
-                  {% else %}
-                    {% include "molecules/cards/card.html" with resource=resource.article show_author=True %}
-                  {% endif %}
-                {% elif type == "person" %}
-                  {% include "molecules/cards/card.html" with resource=resource.person %}
-                {% elif type == "event" %}
-                  {% include "molecules/cards/card.html" with resource=resource.event %}
-                {% endif %}
-              {% empty %}
-                <div class="filter-list-no-results">
-                  <h2>{{ no_resources_message|default:"No results found" }}</h2>
-                </div>
-              {% endfor %}
-            </div>
+      {% empty %}
+        <div class="filter-list-no-results">
+          <h2>{{ no_resources_message|default:"No results found" }}</h2>
+        </div>
```

Comment:
Could we just use the `h2` or do we need the wrapping `div` element?

Reply:
Tidying this up is on my list for the next pass. 👍